src/FixedDataTableBufferedRows.js

```diff
@@ -231,5 +229,15 @@ class FixedDataTableBufferedRows extends React.Component {
       />
     );
   }
+
+  /**
+   * @param {?React.ReactElement} rowA
+   * @param {?React.ReactElement} rowB
+   * @returns {number}
+   * @private
+   */
+  rowSortComparator(rowA, rowB) {
+    return (rowA?.props.ariaRowIndex || -1) - (rowB?.props.ariaRowIndex || -1);
```

Comment:
If the intend here is to use -1 only when ariaRowIndex is null or undefined then please use nullish coalescing (??) instead of ||,

If thats not the case, can `ariaRowIndex` be 0 if yes why we taking value  -1 in that case.

Reply:
> If the intend here is to use -1 only when ariaRowIndex is null or undefined then please use nullish coalescing (??) instead of ||,

Good point.

To answer your other questions, Aria row index can't be zero. 
We can take any value outside the usual row index range as the default.
I purposefully didn't wanna use `Infinity` cause it can behave weirdly with arithmetic operations.

Reply:
> To answer your other questions, Aria row index can't be zero.
Could you comment this somewhere ?

I think `rowA?.props.ariaRowIndex  ?? -1` is still better as the intend is clear in this case.

Reply:
done